src/_scss/pages/agencyV2/overview/_visualizationSection.scss

```diff
@@ -0,0 +1,35 @@
+.visualization-section {
+    text-align: center;
+    height: 100%;
+    @supports not (-ms-high-contrast: none) {
```

Comment:
IE does not correctly handle Flexbox `flex-direction: column`, hence the `@supports` query (which is also incompatible with IE) to selectively apply those styles to bottom-align the visualization labels. This is kind of hacky but it works to get the alignment we want in other browsers while keeping IE from looking terrible.